cryptostore/aggregator/aggregator.py

```diff
@@ -59,6 +59,7 @@ def run(self):
             if len(interval) > 1:
                 multiplier = int(interval[:-1])
                 interval = interval[-1]
+            base_interval = interval
```

Comment:
I'm not sure what you're changing here - base_interval is set to interval, but after its already been changed. It doesnt change after this, so base_interval should == interval

Reply:
interval is modified below here, lines 63-70. If it is ‘M’, ‘H’, ‘D’, it becomes an integer. This is the bug being fixed.